model/schema/highThroughputExpression.yaml

```diff
@@ -0,0 +1,578 @@
+    description: >-
+      The free text describing the amount of sample used
+    domain: HighThroughputExpressionDatasetSampleAnnotation
+    range: string
+
+  agm_biosample:
+    description: >-
+      The AGM biosample referenced in an AGMBioSampleGenomicInformation
+    domain: AGMBioSampleGenomicInformation
+    range: AffectedGenomicModel
+
+  agm_biosample_dto:
+    inlined: true
+    description: >-
+      The AGM biosample referenced in an AGMBioSampleGenomicInformation
+    domain: AGMBioSampleGenomicInformation
+    range: AffectedGenomicModelDTO
+
+  allele_biosample:
+    description: >-
+      The allele biosample referenced in an AlleleBioSampleGenomicInformation
+    domain: AlleleBioSampleGenomicInformation
+    range: Allele
+
+  allele_biosample_dto:
+    inlined: true
+    description: >-
+      The allele biosample referenced in an AlleleBioSampleGenomicInformationDTO
+    domain: AlleleBioSampleGenomicInformationDTO
+    range: AlleleDTO
+
+  assembly_versions: # assemblyVersions in agr_schemas json
+    description: >-
+      single or paired end sequencing; might be better in experiment info
+    domain: HighThroughputExpressionDatasetSampleAnnotation
+    range: string
+    multivalued: true
+
+  biosample:
+    description: >-
+      The biosample referenced in BioSampleGenomicInformation: an AGM (fish, genotype or strain) or allele
+
+  biosample_type:
+    description: >-
+      Indicates the type of biosample; enum: fish, genotype, strain or allele
+    domain: BioSampleGenomicInformation
+    range: string
+
+  biosample_text:
+    description: >-
+      The free text of the biosample genomic information if no biosample is available
+    domain: BioSampleGenomicInformation
+    range: string
+
+  category_tags:
+    description: >-
+      A list of controlled vocabulary terms to describe the experiment
+    domain: HighThroughputExpressionDatasetAnnotation
+    range: VocabularyTerm
+    multivalued: true
+    required: true
+
+  category_tag_names:
+    description: >-
+      A list of controlled vocabulary terms to describe the experiment
+    domain: HighThroughputExpressionDatasetAnnotationDTO
+    range: string
+    multivalued: true
+    required: true
+
+  channel_id: # channelId in agr_schemas json
+    description: >-
+      for microarray experiments, channel ID
+    domain: MicroarraySampleDetails
+    range: string
+    multivalued: false
+
+  channel_number: # channelNum in agr_schemas json
+    description: >-
+      for micorarray data, which channel this sample is in, 1 or 2
+    domain: MicroarraySampleDetails
+    range: integer # enum [1,2]
+    multivalued: false
+
+  dataset_id: # primaryId in agr_schemas htpId.json
+    description: >-
+      The ID from the data provider for the dataset. When available,
+      this is the GEO ID. When no GEO ID is available, it will be the
+      ArrayExpress ID. When these both are unavailable, it will be the
+      MOD ID.
+    range: uriorcurie
```

Comment:
In the old schema this is an object (schema in htpId.json).  Should this really have a range of `HighThroughputExpressionDatasetID`?

Reply:
Yes, fixing now